redfish-core/lib/account_service.hpp

```diff
@@ -994,6 +996,74 @@ inline void
         std::variant<std::string>(userNameAttribute));
 }
 
+// This file holds the value of the AllowUnauthACFUpload property.  An
+// Administrator user can GET and PATCH this property.  Contrast with the
+// readonly Oem.IBM.ACFWindowActive property (in /redfish/v1) which is true
+// when either AllowUnauthACFUpload or the "OpPanel function 74 time window"
+// is active.  Low-level design:
+//  - When this file exists, AllowUnauthACFUpload=True.
+//  - Otherwise, AllowUnauthACFUpload=False (the default value).
+const std::filesystem::path allowUnauthACFUploadFilename{"/var/lib/AllowUnauthACFUpload"};
+
+inline bool SetPropertyAllowUnauthACFUpload(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp, bool allow)
+{
+    if (allow)
+    {
+        // Create the file to indicate the property is true
+        std::ofstream file;
+        file.open(allowUnauthACFUploadFilename);
```

Comment:
I don't see anywhere that we close the file

Reply:
The ofstream object is destructed when it does out of context.  Having the destructor close the file is the recommended usage pattern.  (Should I use some other code to create the file?)  Overall, this code creates/opens the file, change its mode to 600, and closes the file.  (I copy/pasted it from elsewhere in BMCWeb.)

Reply:
No, it's fine. I wasn't sure if it the file was closed on fd destructure.